app/javascript/article-form/components/Help/index.jsx

```diff
@@ -0,0 +1,94 @@
+import { h } from 'preact';
+import { useState } from 'preact/hooks';
+import PropTypes from 'prop-types';
+import { ArticleFormTitle } from './ArticleFormTitle';
+import { TagInput } from './TagInput';
+import { BasicEditor } from './BasicEditor';
+import { EditorFormattingHelp } from './EditorFormattingHelp';
+import { Modal } from '@crayons';
+
+const renderModal = (onClose, title, selector) => {
+  const helpHtml =
+    document.getElementById(selector) &&
+    document.getElementById(selector).innerHTML;
+
+  return (
+    <Modal onClose={onClose} title={title}>
+      <div
+        // eslint-disable-next-line react/no-danger
+        dangerouslySetInnerHTML={{ __html: helpHtml }}
+      />
+    </Modal>
+  );
+};
+
+export const Help = ({ previewShowing, helpFor, helpPosition, version }) => {
+  const [state, setState] = useState({
+    liquidShowing: false,
+    markdownShowing: false,
+    frontmatterShowing: false,
+  });
+
+  const openModal = (varShowing) => {
```

Comment:
```suggestion
  const openModal = (helpSection) => {
```

suggestion (not-blocking) - while we are refactoring these files I think it would be beneficial to improve some of the arg names here to better reflect what's being passed in (same with `closeModal`)